src/main/java/com/rakovets/course/java/core/practice/reflection/ReflectionUtils.java

```diff
@@ -0,0 +1,53 @@
+package com.rakovets.course.java.core.practice.reflection;
+
+import java.lang.reflect.Field;
+import java.lang.reflect.InvocationTargetException;
+import java.lang.reflect.Method;
+import java.util.ArrayList;
+import java.util.Collections;
+import java.util.List;
+
+public class ReflectionUtils {
+
+    public static Field getField(Class<?> clazz, String field) throws NoSuchFieldException {
+        return clazz.getDeclaredField(field);
+    }
```

Comment:
You are passing the exception handling problems to the client. This is bad for utilities.